packages/sequelize-transparent-cache/lib/index.js

```diff
@@ -1,6 +1,9 @@
-const classMethods = require('./methods/class')
+const methods = require('./methods/class')
```

Comment:
it is still methods that available only on class, so let's keep their name as `classMethods`, they should  include `manual` and `auto` properties.